cognite_toolkit/_cdf_tk/commands/repo.py

```diff
@@ -19,6 +20,10 @@
 ]
 
 
+if FeatureFlag.is_enabled(Flags.ADO_PIPELINES):
+    REPOSITORY_HOSTING.insert(1, "Azure DevOps")
```

Comment:
Did not mean to push a commit, tested if I could set suggestion locally and push them. Apparently not :(